packages/zoe/test/unitTests/contracts/test-priceAggregator.js

```diff
@@ -518,13 +518,17 @@ test('oracle invitation', async t => {
   t.deepEqual(value4.quoteAmount.value, makeQuoteValue(9n, 1_234_000n));
 
   updater2.updateState('1234.567890');
+
+  await eventLoopIteration(); // pretend this is a new kernel delivery
```

Comment:
relieved to see that this is the only contract change needed :)

I wonder if we should make the ManualTimer run that await within its `tick()`

Reply:
I tried adding it to `tick()` (i.e. tick already runs it after scheduling wakeups, but I also added it before scheduling them) but that broke a different timing assumption in the documentation repo's coveredCall snippets.

Reply:
Yeah, I've also been tempted, but not found a productive place in manualTimer to add `eventLoopIteration()`. Sometimes it's needed outside the timer in tests.

Reply:
We could have two methods: to run the loop or not. The 'tick' could be whichever is the more common case and the less common case gets a longer name.

Reply:
I prefer leaving this subtlety explicit in the tests rather than having subtly different timer methods.